src/server/api/geo/getBounds.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from 'express'
+
+import { CountryIso } from '@meta/area'
+
+import { GeoController } from '@server/controller/geo'
+import Requests from '@server/utils/requests'
+
+export const getBounds = async (req: Request<never, never, never, { countryIso: CountryIso }>, res: Response) => {
```

Comment:
Do we have a request type already for this?
I don't remember, if you could check

(Request with only countryIso)

Reply:
I didn't see one with only countryIso, should I add it under geo request types?

Reply:
Sure, we can move it to higher level if it is used elsewhere 👍